### PR TITLE
[13.0] [MIG] Account Fiscal Position Rule Sale

### PR DESCRIPTION
**Pull request by ThomasBinsfeld**
Depends on https://github.com/OCA/account-fiscal-rule/pull/100

### CODE REVIEW

**Review by rvalyi — APPROVED** [review]

**Review by chrisdec — APPROVED** [review]
functional test

File: .travis.yml [hunk 1]
```diff
@@ -37,6 +37,10 @@ env:
     - VERSION="13.0" TESTS="0" LINT_CHECK="0" MAKEPOT="0"
 
 install:
+  #  FIXME:
+  - git clone --branch 13.0-mig_account_fiscal_position_rule_tbi
+    https://github.com/acsone/account-fiscal-rule.git PR100
+  - rm -rf account_fiscal_position_rule && cp -ar PR100/account_fiscal_position_rule .
```
Comment (dreispt) [inline .travis.yml]:
```suggestion
```

Comment (dreispt) [reply .travis.yml]: BTW, it's best to not do these changes and just order a TravisCI rebuild once the dependencies are merged. It you went that way, I would have merged this right away.

Comment (ThomasBinsfeld) [reply .travis.yml]: Done, thanks

**Pull request merged**